infra/cmake/packages/Opencl_HeadersConfig.cmake

```diff
@@ -2,19 +2,24 @@ function(_Opencl_Headers_import)
   nnas_find_package(Opencl_HeadersSource QUIET)
 
   # NOTE This line prevents multiple definitions of target
-  if(TARGET Headers)
+  if(TARGET OpenCL_Headers)
     set(Opencl_HeadersSource_DIR ${Opencl_HeadersSource_DIR} PARENT_SCOPE)
     set(Opencl_Headers_FOUND TRUE PARENT_SCOPE)
     return()
-  endif(TARGET Headers)
+  endif(TARGET OpenCL_Headers)
 
   if(NOT Opencl_HeadersSource_FOUND)
     message(STATUS "Opencl_Headers: Source not found")
     set(Opencl_Headers_FOUND FALSE PARENT_SCOPE)
     return()
   endif(NOT Opencl_HeadersSource_FOUND)
 
-  add_extdirectory("${Opencl_HeadersSource_DIR}" OPENCL_HEADERS EXCLUDE_FROM_ALL)
+  # We don't need test builds and installs, we only need headers.
```

Comment:
@Hyunjun85 If changes on opencl_header not depends on this PR, please make another PR for this, first.

Reply:
@chunseoklee We found an issue that I often had to build twice.
And the opencl header has a cycling dependency for that.
So we tried to resolve like above. 

If It is not good for this pr, please let me know once again.